packages/manager/apps/web/client/app/hosting/dashboard/hosting.service.js

```diff
@@ -600,30 +620,35 @@ import union from 'lodash/union';
        * Create a user
        * @param {string} serviceName
        * @param {string} description
+       * @param {number} id
        * @param {string} login
        * @param {string} password
        */
-      userLogsCreate(serviceName, description, login, password) {
-        return this.OvhHttp.post(`/hosting/web/${serviceName}/userLogs`, {
-          rootPath: 'apiv6',
-          data: {
-            description,
-            login,
-            password,
+      userLogsCreate(serviceName, description, id, login, password) {
```

Comment:
```suggestion
      userLogsCreate(serviceName, id, description, login, password) {
```

Reply:
I reordered the params of the function to match apiv6